daemon/openrazer_daemon/dbus_services/dbus_methods/chroma_keyboard.py

```diff
@@ -457,7 +457,7 @@ def set_custom_effect(self):
     Set the device to use custom LED matrix
     """
     # TODO uncomment
```

Comment:
Might want to remove the TODO comment as well since you're uncommenting them.

Reply:
True, but I may end up re-commenting them, so that will be a good reminder.